e2e/etcd_spawn_cov.go

```diff
@@ -31,18 +31,9 @@ import (
 
```

Comment:
`const noOutputLineCount = 2 // cov-enabled binaries emit PASS and coverage count lines`